metrics-runtime/src/common.rs

```diff
@@ -1,58 +1,68 @@
 use crate::data::AtomicWindowedHistogram;
+use metrics_core::{Key, ScopedString};
 use metrics_util::StreamingIntegers;
 use quanta::Clock;
-use std::borrow::Cow;
 use std::ops::Deref;
 use std::sync::atomic::{AtomicI64, AtomicU64, Ordering};
 use std::sync::Arc;
 use std::time::{Duration, Instant};
 
-/// Optimized metric name.
-///
-/// This can either be a [`&'static str`](str) or [`String`].
-pub type MetricName = Cow<'static, str>;
-
 /// A scope, or context, for a metric.
 ///
 /// Not interacted with directly by end users, and only exposed due to a lack of trait method
 /// visbility controls.
```

Comment:
If you would prefer to hide this to users i suggest you use `#[doc(hidden)]` which will remove it from the documentation.

Reply:
Ah yeah, derp! I'll add that. :)

Reply:
All set.